portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase

```diff
@@ -120,13 +120,67 @@ definition {
 	}
 
 	@description = "Verify that a help text can be added"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132461 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Grid");
+
+		var gridFieldName = FormFields.getFieldName();
+
+		FormFields.editGridOptions(
+			gridField = "Rows",
+			rowNumber = "1",
+			optionValue = "Option 1");
+
+		FormFields.editGridOptions(
+			gridField = "Columns",
+			rowNumber = "1",
+			optionValue = "Option 2");
+
+		
+		FormFields.editHelpText(
+			fieldName = "tip",
+			key_fieldLabel = "",
+			fieldHelpText = "Grid Help Text");
+
+		FormViewBuilder.assertHelpTextPresent(
```

Comment:
@evanilson-santana Is it possible to use` FormFields.viewFieldHelpText` instead?